wear/src/main/java/io/homeassistant/companion/android/home/HomePresenterImpl.kt

```diff
@@ -106,4 +106,13 @@ class HomePresenterImpl @Inject constructor(
     override suspend fun setWearHomeFavorites(favorites: List<String>) {
         integrationUseCase.setWearHomeFavorites(favorites.toSet())
     }
+
+    override suspend fun getTileShortcuts(): List<String> {
+        return integrationUseCase.getTileShortcuts()
+    }
+
+    override suspend fun setTileShortcuts(entities: List<String>) {
+        integrationUseCase.setTileShortcuts(entities)
+        view.refreshShortcutsTile()
+    }
```

Comment:
Rather than passing a string we should create a new class to pass with the needed fields.  This will help when the Room DB is ready for wear.

Reply:
I like that idea

Reply:
Implemented it in SimplifiedEntity, let me know what you think.

Also, shall I go ahead to do the same for favorite entities? They currently use the same method